README.md

```diff
@@ -4,4 +4,6 @@ This is one of the prototype repositories for Autoware Core/Universe that AWF ag
 
 Please see [autowarefoundation/autoware](https://github.com/autowarefoundation/autoware) for more details.
 
+Also see [Autoware Universe Documentation](https://github.com/autowarefoundation/autoware.universe).
```

Comment:
How about this? BTW, you can delete the existing sentences.

```md
For Autoware's general documentation, see [Autoware Documentation](https://autowarefoundation.github.io/autoware.universe/).

For detailed documents of Autoware Universe components, see [Autoware Universe Documentation](https://autowarefoundation.github.io/autoware.universe/).
```

Reply:
Oh i mistakenly use wrong link, thank you.